### DISCUSSION TITLE
Missing download link for 4.7.2 on the resteasy.github.io website

### DISCUSSION THREAD

**Discussion by baud001** [resteasy/resteasy#2929] [Q&A] [answered]
Hi;

I'm not sure it's the right location for that, sorry if I'm overstepping.

I was checking the [downloads page](https://resteasy.github.io/downloads/) on the resteasy website and the most recent 4.7 version available for download is 4.71, whereas the most recent version on Maven and Github is 4.7.2. 

Have a nice day.

**Comment by jamezp** [comment, marked as answer]
@baud001 Sorry about that. Should be updated now.

Just so we know how the downloads are being used since we're considering some changes here. What do you use the downloads for and what parts do you download if that is something you can share?

**Reply by baud001** [reply]
Hi,

Thank you. 
We're downloading the zip containing the jars, those jars are then stored in our homegrown dependency management system and then used in our projects that need RESTEasy.
